@vates/otp/index.mjs

```diff
@@ -0,0 +1,111 @@
+import { base32 } from 'rfc4648'
+import { webcrypto } from 'node:crypto'
+
+const { subtle } = webcrypto
```

Comment:
this module seems unmaintained.  
maybe we can  use native crypto on client side if support is good enough ? https://caniuse.com/cryptography